src/NuGet.Clients/NuGet.CommandLine/Commands/Command.cs

```diff
@@ -104,9 +104,17 @@ public void Execute()
             {
                 if (String.IsNullOrEmpty(ConfigFile))
                 {
+                    string configFileName = null;
+
+                    PackCommand packCommand = this as PackCommand;
```

Comment:
Any reason why this is needed? Why is the pack command a special case when other commands are using the config file parameter also?

Reply:
I don't think the other commands have config file working to override the default config file. This was the only way I could get config file working for pack and override the default config files. If there is a better way let me know and I can change it.